pkg/handler/direct_handler.go

```diff
@@ -439,6 +439,13 @@ func (d *DirectHandler) Proxy(asset model.PermAsset) {
 			utils.IgnoreErrWriteString(d.term, lang.T("ACL reject"))
 			utils.IgnoreErrWriteString(d.term, utils.CharNewLine)
 			return
+		case model.ACLFaceVerify:
+			// todo: 需要人脸验证 后续需要发站内信通知用户，并且等待用户人脸验证通过
+			logger.Errorf("Create connect token and auth info failed: %s %s", tokenInfo.Code, tokenInfo.Detail)
+			msg := lang.T("Face verification is not supported yet. Please use the WebTerminal to connect the asset.")
+			utils.IgnoreErrWriteString(d.term, msg)
+			utils.IgnoreErrWriteString(d.term, utils.CharNewLine)
+			return
 		case model.ACLReview:
 			reviewHandler := LoginReviewHandler{
 				readWriter: d.wrapperSess,
```

Comment:
In the given source code snippet, some key lines that may indicate potential issues include the use of "utils.IgnoreErrWriteString" which seems incorrect because error handling using this function often involves logging rather than writing strings back out.

It is recommended to review these specific functions:

- `ignoreError()` and their implementations should correctly log errors instead of simply ignoring them.
  
- Check whether `ignoreError` is part of or imported from necessary packages like `log` for proper context and functionality. It could be causing unnecessary overhead if it's used with other non-standard functions that don't need such logging behavior.

Additionally, I noticed that there might possibly be a syntax/mistake on line 446 in the case structure. The keyword was incorrectly mentioned twice as an argument. Here's the corrected version assuming the intention behind the keywords were correct:

```

case model.ACLReview:
   // todo: Reviewing the content and asking users for feedback...
   logger.Errorf("Failed to create connect token and auth information after reviewing: %s %s", tokenInfo.Code, tokenInfo.Detail)
   msg := fmt.Sprintf(lang.T("Reviewing asset content..."), asset.PermStr())
   utils.WriteStringToConsole(d.term, msg)
   return
```  

If the purpose of checking the review status against different statuses wasn’t intended, the logic should be adjusted accordingly within the respective cases. This ensures clarity, precision, and avoids unexpected behaviors during debug analysis.